okhttp/src/main/java/io/grpc/transport/okhttp/OkHttpClientTransport.java

```diff
@@ -161,6 +165,9 @@
   @GuardedBy("lock")
   private LinkedList<PendingStream> pendingStreams = new LinkedList<PendingStream>();
   private final ConnectionSpec connectionSpec;
+  private FrameWriter rawFrameWriter;
```

Comment:
Name this `testFrameWriter` and don't bother setting it normally?

That also makes it easier to do rawFrameWriter.connectionPreface(); immediately after the connection establishes instead of after you deal with `lock` in `start()`